web/client/utils/styleparser/StyleParserUtils.js

```diff
@@ -613,14 +613,14 @@ export const drawWellKnownNameImageFromSymbolizer = (symbolizer) => {
         && !(symbolizer.wellKnownName || '').includes('shape://');
     const canvas = document.createElement('canvas');
     const ctx = canvas.getContext('2d');
-    const radius = symbolizer.radius;
+    const radius = (symbolizer.radius || symbolizer?.size / 2);
```

Comment:
This is changing how the `drawWellKnownNameImageFromSymbolizer` used mainly for markers.
The PrintStyleParser was using the [drawWellKnownNameImageFromSymbolizer](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/styleparser/PrintStyleParser.js#L88) to quickly access the imagesCache to get the original width and height of the image so if the function is reaching this point means that the id computed by getImageIdFromSymbolizer(symbolizer) is not correct.
The reason is related to the introduction of the style with properties functionality (https://github.com/geosolutions-it/MapStore2/pull/9541) that introduces some modifications to the symbolizer.
This parsing issue has been partially solved here #9776 but probably the part related to print was missing.
Here the proposed solution:
- create a new function in the StyleParserUtils called getCachedImageById
- replace the drawWellKnownNameImageFromSymbolizer with getCachedImageById inside the Icon section of PrintStyleParser
- Add to the symbolizerToPrintMSStyle function a new argument that is the originalSymbolizer (not parsed symbolizer)
- Pass the not parsed symbolizer as argument of getCachedImageById
- use the symbolizer size as width and height as a fallback only if getCachedImageById returns undefined 

```js
export const getCachedImageById = (symbolizer) => {
    const id = getImageIdFromSymbolizer(symbolizer);
    return imagesCache[id];
}
```